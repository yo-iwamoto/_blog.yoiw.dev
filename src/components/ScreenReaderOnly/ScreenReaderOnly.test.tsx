import { ScreenReaderOnly } from '.';
import { render } from '@testing-library/react';

describe('ScreenReaderOnly', () => {
  it('正しく表示されること', () => {
    const { getByText, asFragment } = render(<ScreenReaderOnly>help text</ScreenReaderOnly>);

    expect(getByText('help text')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
