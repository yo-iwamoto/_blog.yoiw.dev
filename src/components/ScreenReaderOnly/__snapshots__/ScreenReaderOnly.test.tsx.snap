// Vitest Snapshot v1

exports[`ScreenReaderOnly > 正しく表示されること 1`] = `
<DocumentFragment>
  <span
    class="sr-only"
  >
    help text
  </span>
</DocumentFragment>
`;
